suitcase/pizza_box/handlers.py

```diff
@@ -0,0 +1,162 @@
+import pandas as pd
+import os
+
+'''
+These handlers read from data generated by the PIZZA boxes at NSLS-II.
+There are currently two types: analog and encoder pizza boxes.
+
+The analog pizza boxes have the form:
+timestamp (s)    timestamp (ns)    index    value1 value2
+where 
```

Comment:
some of these lines have trailing whitespace. can you remove them? use flake8 to find them.
here are the lines:
```
./suitcase/pizza_box/handlers.py:10:6: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:14:55: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:16:58: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:17:46: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:23:57: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:31:51: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:33:44: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:34:55: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:35:58: W291 trailing whitespace
./suitcase/pizza_box/handlers.py:38:59: W291 trailing whitespace
```

thanks!